### PR TITLE
hooks: support concurrent tests run

### PR DESCRIPTION
**Pull request by viktor-kurchenko**
The Hooks.AddConnectivityTests method signature changed to support concurrent test run.

### CODE REVIEW

File: cilium-cli/api/api.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ var _ Hooks = &NopHooks{}
 func (*NopHooks) AddSysdumpFlags(*pflag.FlagSet)                                  {}
 func (*NopHooks) AddSysdumpTasks(*sysdump.Collector) error                        { return nil }
 func (*NopHooks) AddConnectivityTestFlags(*pflag.FlagSet)                         {}
-func (*NopHooks) AddConnectivityTests(*check.ConnectivityTest) error              { return nil }
+func (*NopHooks) AddConnectivityTests([]*check.ConnectivityTest) error            { return nil }
```
Comment (christarazi) [inline cilium-cli/api/api.go]: Can we turn this into a variadic variable instead? That we can pass either a single connectivity test or multiple without having to initialize a slice variable.

**Review by michi-covalent — APPROVED** [review]
lgtm please address the feedback from chris before merging 👍

File: cilium-cli/api/api.go (hunk 1 above)
Comment (viktor-kurchenko) [reply cilium-cli/api/api.go]: Fixed, thank you!

**Review by christarazi — APPROVED** [review]

**Pull request merged**